src/Drawer/Drawer.js

```diff
@@ -197,8 +199,16 @@ class Drawer extends Component {
     }
   };
 
+  getTranslatedWidth() {
+    if (this.props.width <= 1) {
+      return this.props.width * window.innerWidth;
```

Comment:
That's gonna break server-side rendering where `window.innerWidth` is undefined.
Option 1. Add a guard condition
Option 2. Use pourcentage (`100%`).